docker-compose.dev.yaml

```diff
@@ -12,7 +12,6 @@ services:
       - ./graphql/hasura/metadata/versions/latest:/hasura-metadata
   postgres:
     image: postgres:15.2-alpine
-    container_name: postgres
```

Comment:
Will this then not cause a port clash?

Reply:
If you have two containers running at the same time. Yes. But this way allows you to stop conflicting containers.

If the name is the same, then you have to not only stop it, but remove the container to resolve the naming clash. So it can be very frustrating working on multiple projects with the same name for the container.

Reply:
Gotcha, might want to just note it somewhere. Like a table listing required open ports for local dev.

Reply:
Added some ports to the Wiki